From 3534e99d18952fc87f98cf60eb7767374a1371c4 Mon Sep 17 00:00:00 2001
From: Tomas Chvatal <tchvatal@suse.cz>
Date: Sun, 29 Apr 2012 10:45:59 +0200
Subject: [PATCH] Fix buildissue in svtools with missing include.

cpp: line 33, Fatal error: Cannot open include file
"svtools/svtools.hrc"

Change-Id: Ic7bd1af8f14cc0e44328c9f149c5dc45f04e5afb
---
 svtools/AllLangResTarget_svt.mk |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/svtools/AllLangResTarget_svt.mk b/svtools/AllLangResTarget_svt.mk
index 430eb23..4a4421a 100644
--- a/svtools/AllLangResTarget_svt.mk
+++ b/svtools/AllLangResTarget_svt.mk
@@ -39,7 +39,8 @@
 $(eval $(call gb_SrsTarget_set_include,svt/res,\
     $$(INCLUDE) \
     -I$(WORKDIR)/inc \
-    -I$(realpath $(SRCDIR)/svtools/source/inc) \
+    -I$(SRCDIR)/svtools/source/inc \
+    -I$(SRCDIR)/svtools/inc \
 ))
 
 $(eval $(call gb_SrsTarget_add_files,svt/res,\
